src/main/scala/org/broadinstitute/dsde/workbench/avram/api/AvramRoutes.scala

```diff
@@ -20,6 +22,10 @@ case class DbPoolStats(@BeanProperty numActive: Int, @BeanProperty numIdle: Int,
 class AvramRoutes {
 
   private val log = Logger.getLogger(getClass.getName)
+  val configFactory = ConfigFactory.parseResources("app.conf") //s.withFallback(ConfigFactory.load())
```

Comment:
We should extract this into a `Config` object so that individual components don't have to repeat the knowledge of where to read configs from.